Buildings/Fluid/HeatExchangers/ThermalWheels/Latent/BaseClasses/PartialWheel.mo

```diff
@@ -0,0 +1,231 @@
+  parameter Real P_nominal(final unit="W")
+    "Power consumption at the design condition"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.Efficiency epsSenCoo_nominal(
+    final max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the cooling mode"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.Efficiency epsLatCoo_nominal(
+    final max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the cooling mode"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.Efficiency epsSenHea_nominal(
+    final max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the heating mode"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.Efficiency epsLatHea_nominal(
+    final max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the heating mode"
+    annotation (Dialog(group="Nominal condition"));
+
+  parameter Modelica.Units.SI.Efficiency epsSenCooPL(
+    final max=1) = 0.75
+    "Part load (75%) sensible heat exchanger effectiveness at the cooling mode"
+    annotation (Dialog(group="Part load effectiveness"));
+  parameter Modelica.Units.SI.Efficiency epsLatCooPL(
+    final max=1) = 0.75
+    "Part load (75%) latent heat exchanger effectiveness at the cooling mode"
+    annotation (Dialog(group="Part load effectiveness"));
+  parameter Modelica.Units.SI.Efficiency epsSenHeaPL(
+    final max=1) = 0.75
+    "Part load (75%) sensible heat exchanger effectiveness at the heating mode"
+    annotation (Dialog(group="Part load effectiveness"));
+  parameter Modelica.Units.SI.Efficiency epsLatHeaPL(
+    final max=1) = 0.75
+    "Part load (75%) latent heat exchanger effectiveness at the heating mode"
+    annotation (Dialog(group="Part load effectiveness"));
+
+  Modelica.Blocks.Interfaces.RealOutput P(
+    final unit="W")
+    "Electric power consumption"
+    annotation (Placement(transformation(extent={{100,-10},{120,10}})));
+  Buildings.Fluid.HeatExchangers.ThermalWheels.Latent.BaseClasses.HeatExchangerWithInputEffectiveness hex(
+    redeclare package Medium1 = Medium1,
+    redeclare package Medium2 = Medium2,
+    final m1_flow_nominal=m1_flow_nominal,
+    final m2_flow_nominal=m2_flow_nominal,
+    final show_T=true,
+    final dp1_nominal=dp1_nominal,
+    final dp2_nominal=dp2_nominal)
+    "Heat exchanger"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
+  Buildings.Fluid.HeatExchangers.ThermalWheels.Latent.BaseClasses.Effectiveness effCal(
+    final epsSenCoo_nominal=epsSenCoo_nominal,
+    final epsLatCoo_nominal=epsLatCoo_nominal,
+    final epsSenCooPL=epsSenCooPL,
+    final epsLatCooPL=epsLatCooPL,
+    final epsSenHea_nominal=epsSenHea_nominal,
+    final epsLatHea_nominal=epsLatHea_nominal,
+    final epsSenHeaPL=epsSenHeaPL,
+    final epsLatHeaPL=epsLatHeaPL,
+    final VSup_flow_nominal=m1_flow_nominal/1.293)
```

Comment:
Compute the density from the medium properties, as a parameter, and use that value, as we also do for `cp_default`